bin/scripts/config-helper.ts

```diff
@@ -0,0 +1,157 @@
+        ],
+        message: "Select a configuration type:",
+    })
+
+    if(advancedOrBasicResponse.configType === advanced) {
+        await advancedPrompts();
+        console.log(questions.length)
+    }
+
+
+    // The final list of questions has been created, this answer object will be the final config JSON
+    const answers = await prompt(questions);
+    
+      
+    console.log(`Response: ${JSON.stringify(answers)}`); 
+
+    fs.writeFileSync('./lib/config.json', JSON.stringify(answers, undefined, 4));
+}
+
+
+async function advancedPrompts() {
+    const vpcResponse = await prompt({
+        type: "confirm",
+        name: "existingVpc",
+        message: "Do you want to use existing VPC? (selecting false will create a new VPC for MLSpace)",
+    })
+    if(vpcResponse.existingVpc) {
+        const vpcQuestions = [
+            {
+                type: "input",
+                name: "EXISTING_VPC_NAME",
+                message: "If MLSpace is being deployed into an existing VPC this should be the name of that VPC",
+            },
+            {
+                type: "input",
+                name: "EXISTING_VPC_ID",
+                message: "If MLSpace is being deployed into an existing VPC this should be the ID of that VPC",
+            },
+            {
+                type: "input",
+                name: "EXISTING_VPC_DEFAULT_SECURITY_GROUP",
+                message: "If MLSpace is being deployed into an existing VPC this should be the default security group of that VPC",
+            },
+        ]
+        questions.push(...vpcQuestions);
+    }
+
+    const roleResponse = await prompt({
+        type: "confirm",
+        name: "existingRoles",
+        message: "Do you want to use existing IAM Roles? (selecting false will create new IAM Roles for MLSpace actions)",
+    });
+    if(roleResponse.existingRoles) {
+        const roleQuestions = [
+            {
+                type: "input",
+                name: "S3_READER_ROLE_ARN",
+                message: "Arn of an existing IAM role to use for reading from the static website S3 bucket. If not specified a new role with the correct privileges will be created",
+            },
+            {
+                type: "input",
+                name: "BUCKET_DEPLOYMENT_ROLE_ARN",
+                message: "Arn of an existing IAM role to use for deploying to the static website S3 bucket. If not specified a new role with the correct privileges will be created",
+            },
+            {
+                type: "input",
+                name: "NOTEBOOK_ROLE_ARN",
+                message: "Arn of an existing IAM role to associate with all notebooks created in MLSpace. If using dynamic roles based on project/user combinations the specific combination role will be used instead. This value must be set to an existing role because the default CDK deployment will not create one.",
```

Comment:
Should we require this regardless then? Seems confusing that you can skip this advanced step and assume that this role would be created with the basic setup

Reply:
This description was confusing to me (taken from the README) since it's empty by default in `constants.ts` and I'm able to deploy my stack without issue leaving it empty.